airbyte-cdk/bulk/build.gradle

```diff
@@ -5,14 +5,8 @@ import javax.xml.xpath.XPathConstants
 import javax.xml.xpath.XPathFactory
 import org.w3c.dom.Document
 
-final var cdkVersion = {
-    var props = new Properties()
-    file("core/base/src/main/resources/version.properties").withInputStream(props::load)
-    return props.getProperty('version', 'undefined')
-}()
-
 allprojects {
-    version = cdkVersion
+    version = "0.1.28"
```

Comment:
@edgao @benmoriceau 
I reverted this change as it broke `./gradlew :airbyte-cdk:bulk:checkBuildNumber`.

Reply:
Actually this was broken and Ed's change uncovered it:
The result of xpath search of `"/metadata/versioning/latest"` is undefined